cdk/package-lock.json

```diff
@@ -7088,7 +7088,7 @@
       }
     },
     "aws4embeddedlinux-cdk-lib": {
-      "version": "git+ssh://git@github.com/aws4embeddedlinux/aws4embeddedlinux-ci.git#15aa7c5c329203bc3a10abcb811dedb02f001790",
```

Comment:
Are we sure we want to do it this way? Conceptually, versions of direct dependencies should be present first in package.json, whereas the package-lock.json file should ensure deterministic installations by locking exact versions of dependencies (direct and transient). I would prefer to see some information regarding the version (we can still discuss if commit, or tag or what else) in the package json. Unless there is a good reason not to do it which I am missing?

Reply:
This is the only solution unless we tag.